.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest an idea or improvement
+labels: enhancement
+
+---
+
+<!--
+## Before reporting
+ - Search for the suggestion here. It's possible someone's suggested it before!
+-->
+
+## Useful information to include:
+ - Explanation of how the feature/change should work.
+ - Some rationale/use case for a feature. I'd like to keep Plethora as minimal as possible, so I like have a solid justification for each feature.
```

Comment:
>  I'd like to keep Plethora as minimal as possible

I'm less sure about this than in CC:T. I definitely don't want to bloat the mod, but it's also always been the "well, whatever" mod where all sorts of random stuff ends up.

Maybe something like  "Everything takes time to implement, so I like to have a solid justification for each feature."? Though "It'd be fun" is sometimes justification enough.